otter/effect_dispatcher.py

```diff
@@ -84,3 +86,15 @@ def get_cql_dispatcher(reactor, connection):
             CQLQueryExecute: partial(perform_cql_query, connection)
         })
     ])
+
+
+def get_eviction_dispatcher(supervisor):
```

Comment:
To follow the model elsewhere I'd suggest this be moved to worker_intents.py. I think it's fine to leave the name the same for now since we don't know what else would go in here yet, we can rename it later if we decide to add more worker intents to it.